src/components/CartList/CartList.tsx

```diff
@@ -0,0 +1,25 @@
+import Text from '../common/Text/Text';
+import { CartListCheckboxStyle, CartListHeaderStyle } from './CartList.styles';
+import { TEXT } from '../../constants/text';
+
+function CartListHeader({
+  allSelected,
+  onAllSelectChange,
+}: {
+  allSelected: boolean;
+  onAllSelectChange: () => void;
+}) {
+  return (
+    <div css={CartListHeaderStyle}>
+      <input
+        css={CartListCheckboxStyle}
+        type="checkbox"
+        checked={allSelected}
+        onChange={onAllSelectChange}
+      />
+      <Text varient="caption">{TEXT.ALL_SELECT}</Text>
+    </div>
+  );
+}
```

Comment:
제가 잘못된 피드백을 남겼었네요 ㅠㅠ
기존처럼 children 방식으로 사용하면 CartList가 렌더링 되더라도 CartItem은 children으로 넣어주고 있어서 따로 영향이 없을 것 같아요 ㅎㅎ
CartPage가 렌더링 될 때 CartItem 도 전부다 렌더링 되는걸 이야기 했어야 하는데.. 제가 잘못된 키워드로 남겼었네요 ㅠㅠ
죄송합니다 해삐..!
```suggestion
function CartListHeader({
  allSelected,
  onAllSelectChange,
}: {
  allSelected: boolean;
  onAllSelectChange: () => void;
}) {
  return (
    <div css={CartListHeaderStyle}>
      <input
        css={CartListCheckboxStyle}
        type="checkbox"
        checked={allSelected}
        onChange={onAllSelectChange}
      />
      <Text varient="caption">{TEXT.ALL_SELECT}</Text>
    </div>
  );
}
```

Reply:
여튼 이 부분은 원복 해도 좋아보입니다 ㅎㅎ